docs/guidelines/FAQ/pre-commit-faq.md

```diff
@@ -25,8 +25,9 @@ Pre-commit can be installed in two ways, using Python pip or Homebrew.
 |---|---|
 | `pip install pre-commit` | `brew install pre-commit` |
 
-Once pre-commit is installed you need to manually add it to PATH (user variables). 
-Set up the git hook scripts by running `pre-commit install`. Now pre-commit wil automatically run on git commit!
+Once pre-commit is installed you need to set up the git hook scripts by running `pre-commit install`. Now pre-commit wil automatically run on git commit!
+
+###### NOTE: If `pre-commit` is not available after install, it might be needed to manually add it to PATH (user variables).
```

Comment:
Good work. A super tiny comment; the "(user variables)" is a Windows specific thing and may be deleted? PATH should be sufficient.....

Reply:
Thanks! Fixing that in next commit.